samples/dotnet/clientcreds/Lenus.Samples.ClientCredentialsFlow/src/Services/Agency/ResponseHandler.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Lenus.Samples.ClientCredentialsFlow.Services.Agency.Authentication
+{
+    class ResponseHandler : DelegatingHandler
```

Comment:
Needs an access modifier e.g. public

Reply:
```suggestion
    public class ResponseHandler : DelegatingHandler
```

Reply:
This was a class I ended up not using so have removed the file